### ISSUE TITLE
Using Trinity FASTA file as input for FEELnc_filter.pl

### ISSUE THREAD

**Issue by CristinaOsu** [tderrien/FEELnc#31]
Dear all,

I am trying to run the entire pipeline of FEELnc using Trinity transcript FASTA file as input (instead of cufflinks/stringtie transcripts.GTF). However, I only found possible to run FEELnc_codpot.pl with this type of input but not FEELnc_filter.pl. Is that correct? why is not available the filter step for Trinity transcript FASTA file input?

Thank you so much in advance,

Cristina Osuna

**Comment by tderrien** [comment]
Dear Cristina,

Yes, you are correct.
Using FASTA files as input, FEELnc cannot filter transcripts based on the number of exons for instance (1st module filter.pl) nor classify them wrt closest genes (3rd modules classifier.pl).
As stated in the [introduction](https://github.com/tderrien/FEELnc#introduction), the 1st and 3rd modules require a reference genome while people using Trinity generally don't have reference genome.
HTH

Thomas

**Comment by CristinaOsu** [comment]
Dear Thomas,

Thanks a lot for your quick reply! I do have reference genome but what I ran was the Genome-guided Trinity De novo Transcriptome Assembly (https://github.com/trinityrnaseq/trinityrnaseq/wiki/Genome-Guided-Trinity-Transcriptome-Assembly). I used these transcripts to update my protein coding gene annotation with PASA pipeline (https://github.com/PASApipeline/PASApipeline/wiki/PASA_genome_annotation). I was hoping to use now FEELnc pipeline to include lncRNA annotation as well, but I found weird not being able to filter my transcript FASTA file prior calculating coding potential. How would you recommend proceeding then? I was thinking of mapping my TRINITY transcripts against my genome using gmap or so, and then using the resulting gtf file as input... but I am not sure if this would be a good idea or if it is better start over using e.g Stringtie.
Thank you so much in advance.

All my best,

Cristina Osuna

**Comment by tderrien** [comment]
Yes, in this case, it would be better to map your FASTA assembled transcripts (with gmap or minimap...) onto you reference genome so that you would be able to use the entire FEELnc pipeline.
Actually, using mapping-first versus assembly-first strategy really depends on the quality of your genome assembly.

Best,

Thomas
